app/views/email-02.html

```diff
@@ -27,11 +27,15 @@ <h2 class="govuk-heading-m">
       </p>
 
       <p>
-      We'll email you when the countersigned agreement is available.
+      If these details do not look correct, contact the Crown Commercial Service at cloud_digital@crowncommercial.gov.uk
       </p>
 
       <p>
-      You'll need to wait for CCS to countersign your agreement before you can sell services.
+      We'll email you when the countersigned agreement is available (this is normally within 3 days).
+      </p>
+
+      <p>
+      You'll need to wait for CCS to countersign your agreement before you can sell services from 28 September 2020.
       </p>
```

Comment:
Please can we amend this:

Once you have a countersigned agreement, you can sell services from Monday 28 September 2020.

Reply:
Done